cli/spec/kontena/command_spec.rb

```diff
@@ -0,0 +1,55 @@
+require_relative '../spec_helper'
+require 'kontena/command'
+
+describe Kontena do
+  let(:subject) do
+    class TestCommand < Kontena::Command
```

Comment:
OTOH I guess this test case is sensitive to the `class.name`...? I'm not sure how that goes, I've seen it have some really weird magic where it picks up the variable name from `foo_class = Class.new ...`..